src/server.test.js

```diff
@@ -16,6 +16,7 @@ describe('Testing the server', () => {
   })
   test('getApp should return an express server after startServer has been called', async () => {
     express.mockReturnValue({
+      set:jest.fn(),
```

Comment:
Try to do what hoie did, he has a better practice to mock express once at the top of your file so your code is cleaner and easier to read.

```
const app = {
  set: jest.fn(),
  use: jest.fn(),
  post: jest.fn(),
  get: jest.fn()
}

express.mockReturnValue(app)

```

Reply:
Yea I was changing my code to meet the code in upstream server.js file, since I thought that was merged one. Will change back after Hoie’s code got merged.